draft-secure-credential-transfer.md

```diff
@@ -772,6 +772,8 @@ in the "Permanent Message Header Field Names" <[](https://www.iana.org/assignmen
     | Header Field Name  | Protocol | Status |   Reference   |
     +--------------------+----------+--------+---------------+
     | Mailbox-Request-ID |   http   |  std   | This document |
+    | Device-Claim       |   http   |  std   | This document |
+    | Device-Attestation |   http   |  std   |This document  |
```

Comment:
Should we prefix these with something Tigress specific?

Reply:
like Mailbox-Device-Claim and Mailbox-Device-Attestation?

Reply:
Would also vote for introducing the "Mailbox-" prefix as proposed by you, @dimmyvi. But please keep in mind: This would require changing "deviceClaim (String, UUID, Required)" in the draft at several places and a change of the corresponding CCC CR.

Reply:
@dimmyvi: I recommend to change "deviceClaim (String, UUID, Required)" to "Mailbox-Device-Claim (String, UUID, Required)" as well (same for deviceAttestation). Otherwise it is ambiguous and might be misleading

Reply:
Corrected all deviceClaim and deviceAttestation headers

Reply:
Great, thx! It seems that you've missed one occurence: "The value of DeviceAttestation header parameter ..."

Reply:
Fixed it. Thank you Manuel!